minecraft_servers/reberion/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "reberion",
+  "nice_name": "Reberion Network",
+  "direct_ip": "reberion.eu",
+  "server_wildcards": [
+    "%.reberion.eu"
+  ],
+  "social": {
+    "web": "https://www.reberion.eu",
+    "twitter": "ReberionFeeds",
+    "teamspeak": "ts.reberion.eu",
+    "discord": "https://reberion.eu/discord",

```

Comment:
As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url